frontend/uv-poc/src/app/order/order.component.ts

```diff
@@ -35,24 +24,12 @@ export class OrderComponent implements OnInit {
 
     this.orderDetailService.get(this.transactionId).subscribe(
       (res: any) => {
-        this.setValues(res['orderDetail']);
+        this.orderData = res['orderDetail'];
+        console.log(this.orderData);
```

Comment:
No `console.log` statements, please

Reply:
sorry @KartikyaThakur

Reply:
Don't be sorry, just remove the `console.log` statement